core/src/components/range/range.vars.scss

```diff
@@ -0,0 +1,5 @@
+/// @prop - Top margin of range's label when in an item
+$range-item-label-margin-top: 10px !default;
+
+/// @prop - Bottom margin of range's label when in an item
+$range-item-label-margin-bottom: 10px !default;
```

Comment:
I know that this is the default but it does seems a bit too much when comparing it to the other labels.
![Screenshot 2023-09-13 at 9 46 02 AM](https://github.com/ionic-team/ionic-framework/assets/13530427/5efe9049-17a6-468c-98b9-089d691f3c9b)

Should we aim to keep it visually similar or stick with the default value?

Reply:
Good point. I'll try removing the bottom margin and see how that looks.

Reply:
Ok how does https://github.com/ionic-team/ionic-framework/pull/28161/commits/db20bd1b8e705144b6d4db0e8c0c98ad4a328d26 look?

Reply:
That looks better. I can't tell if my eyes are playing tricks but it seems that the stacked spacing is less compared to a non-stacked label.
![Screenshot 2023-09-13 at 10 35 49 AM](https://github.com/ionic-team/ionic-framework/assets/13530427/40d324ec-a5b9-493f-88f6-65652feb00de)

Using `3px` seems closer. I'm not a stickler on this though.

Reply:
When using a non-stacked label the label-text-wrapper element adds 10px of bottom margin which is why there's a difference. The same difference exists with the other form controls too, so I don't think that's something that should be handled in this PR.